dynamic_rest/serializers.py

```diff
@@ -83,6 +84,16 @@ def __new__(cls, *args, **kwargs):
             meta = type('Meta', (), {})
             cls.Meta = meta
         meta.list_serializer_class = DynamicListSerializer
+
+        # Check for bulk resource creation with help of resource plural name.
+        # This should happen before actual constructor because class needs to
+        # be changed on DynamicListSerializer.
+        plural_name = cls.get_plural_name()
+        data = kwargs.get('data', fields.empty)
+        if data is not fields.empty and plural_name in data and len(data) == 1:
```

Comment:
OK, so I removed `ENABLE_BULK_CREATION` from settings and make it default behaviour. I also moved the logic from `__new__` to `DynamicModelViewSet.create`.